src/main/java/org/mapfish/print/config/HostMatcher.java

```diff
@@ -19,6 +19,9 @@
 
 package org.mapfish.print.config;
 
+import com.fasterxml.jackson.annotation.JsonSubTypes;
+import com.fasterxml.jackson.annotation.JsonTypeInfo;
```

Comment:
it would be good to add Jackson-annotations to the dependencies and not rely on transitive dependencies

Reply:
Done thanks.